files/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>WoMakers Code</title>
+</head>
+<body>
+	<nav class="wm-nav">
+		<h1 class="wm-nav__logo"></h1>
+		<ul class="wm-nav__menu">
+			<li class="wm-nav__item"><a href="#">Home</a></li>
+			<li class="wm-nav__item"><a href="#">Serviços</a></li>
+			<li class="wm-nav__item"><a href="#"></a>Portfolio</a></li>
+			<li class="wm-nav__item"><a href="#">Sobre Mim</a></li>
+			<li class="wm-nav__item"><a href="#">Contato</a></li>
+		</ul>
+	</nav>
+	<header class="wm-header">
+		<h2 class="wm-header__title">Bem Vind@</h2>
+		<h3 class="wm-header__subtitle">Carabitur iaculis</h3>
+		<p class="wm-header__text">texto texto texto</p>
+	</header>
+	<div class="wm-servicos" id="servicos">
+		<h1 class="wm-servicos__title">Serviços</h1>
+		<h3 class="wm-servicos__subtitle">texto texto texto</h3>
```

Comment:
Tente seguir o padrão de h1, h2, h3

```suggestion
<h2 class="wm-servicos__subtitle">texto texto texto</h2>
```